app/pages/AcountLogin/index.js

```diff
@@ -64,18 +67,20 @@ export default class AccountLogin extends Component {
             onChange={(i) => this.setState({chosenWallet: i})}
           />
           <div>
-            <input
-              className={c('login_password_input', {
-                'login_password_input--error': showError,
-              })}
-              type="password"
-              placeholder="Your password"
-              onChange={e =>
-                this.setState({passphrase: e.target.value, showError: false})
-              }
-              value={passphrase}
-              autoFocus
-            />
+            {currWallet !== 'Ledger' && (
```

Comment:
hm, interesting - so a Ledger wallet can't be locked for privacy? I wonder if we can add a passphrase to the unused `primary` wallet in this case just to have something? Actually, I believe a watch-only wallet can still be locked / unlocked. Even watch only wallets have a master private key that can be encrypted. The key isn't used for addresses but it is used for generating auth tokens.